### DISCUSSION TITLE
✅ Creating an OSPO Local Meetup - Japan (Japanese speaking)

### DISCUSSION THREAD

**Discussion by kuwatm** [todogroup/ospology#223] [Community Proposals]
When I participate Open Source Summit Japan 2022 and communicate with some OSPOers, then I felt that many OSPOers in Japan are struggling to get understanding from colleagues how important OSS use strategically.

I’ve joined OSPO subgroup around members who are in the Open Chain, and we are planning to have meetings regular basis.

On the other hand, TODO Group publishes excellent "OSPO Guides", so I'd also like to connect with people who want to share or want to know best practices in our local community.

In fact, I want many Japanese companies to be able to climb the "Four Stages of the OSPO", and more many engineers get involved in the Open Source community.

**Comment by anajsana** [comment]
Thanks for submitting this request @kuwatm ! From my understanding and based on what we discussed in previous conversations with the OpenChain Japan WG community, the potential milestones for the Japanese OSPO Local Meetup would be:

* Build bridges between OpenChain Japan WG OSPO SIG and the wider TODO / OSPO Community.
* Co-create open source content with regional and global OSPO practitioners
* Create new or adapt existing OSPO resources for Japanese organizations
* Bring discussions about the status of OSPOs in Japan to the TODO community
* Contribute to a Japanese version of the existing OSPO Model 

Moreover, the OSPO Local Japan Organizer would be you (@kuwatm) and @hfukuchi, correct?

Please let me know if this info is correct or if is there anything you'd like to add / change. I'll share this info with the TODO Steering Committee for review and will get back to you ASAP. 🙂

**Reply by kuwatm** [reply]
Thanks for the potential milestones, @anajsana . My understanding is currently same, and no additional information and correction.
We are discussing relationship or structure of TODO local meetup and OpenChain OSPO Subgroup. If I have any news, I will keep you updated.
Regarding Organizer, yes, I prefer that @hfukuchi  and I are co-organizers.  In our side, we’ve agreed about this.

**Comment by anajsana** [comment]
Good News @kuwatm review process has been completed by SC and the `OSPO Local Meetup - Japan (Japanese Language)` proposal is approved! Below are essential details for you to know:

## Registration Platform Acess
- [ ] The next step @anajsana needs to do is to create an `OSPO Local Meetup - Japan` chapter via LF Online Community Platform and give @kuwatm and @hfukuchi chapter admin access to edit and manage attendees.

## Real-time Communication Slack
Also, we usually create a new slack channel in the TODO slack for new Local Chapters, but I know OpenChain Japan WG has an active OSPO SIG channel, so please let me know if you'd like to either (1) have a new `#chapter-japan` in TODO Group Slack or (2) keep the main communication in the OpenChain Japan WG OSPO SIG.

## OSPO Local Meetup Logo & Banner Desing

- [ ] @anajsana will send the OSPOChi version for the OSPO Local Meetup Japan (OSPOlogy / TODO Mascot) and design templates to use on the registration page and SNS promotion

## Meetup Summaries & Documentation

OSPO Local organizers should write a summary of what is being done / accomplished during these meetups on a quarterly basis via OSPOlogy GH Discussions. An example can be found at: https://github.com/todogroup/ospology/discussions/212

**Reply by kuwatm** [reply]
Thanks for this great news! @anajsana 
I will share this information in OpenChain OSPO SIG soon. And I will plan detail.

**Reply by kuwatm** [reply]
Thanks again, I received an email about Platform. @anajsana 
Regarding “Registration Platform Access”, I received an email and I can access below now.
[LF Online Community OSPO Local Meetup - Japan (Japanese Speaking) (linuxfoundation.org)](https://community.linuxfoundation.org/ospo-local-meetup-japan-japanese-speaking/)
I added my picture in my LF account, but currently it is not shown on the Community platform.

Regarding “Real-time Communication Slack”, we are discussing in OpenChain OSPO-sg and voting on Slack until this Thursday, but probably I would ask you to create TODO #chapter-Japan in TODO Group Slack.

**Reply by kuwatm** [reply]
@anajsana 
Sorry for this late update.
May I ask you a new #chapter-Japan on TODO Group Slack?

**Reply by anajsana** [reply]
Sure! it has been created under `#chapter-japan` 👍 

I'm sorry I haven't finished with the design templates and OSPOChi version for the OSPO Local Meetup Japan. Planning to have them ready next week!

> I added my picture in my LF account, but currently it is not shown on the Community platform.

It looks like adding images is disabled by default and neither I can't make changes as general admin. I've contacted the support team to see if we can fix it!

The registration page can be found: https://community.linuxfoundation.org/ospo-local-meetup-japan-japanese-speaking/

![Screenshot 2023-01-26 at 16-41-36 LF Online Community](https://user-images.githubusercontent.com/43671777/214880327-a8b3a085-db34-439d-ad95-10492efa2905.png)

**Reply by kuwatm** [reply]
Thanks for the creation of #chapter-japan.
It has already 5 members (even Japan time is still 9 in the morning).
Also thanks for contacting support team for adding image.
I'm looking forward to receiving OSPOChi version next week!
